llvm-general/test/LLVM/General/Test/Metadata.hs

```diff
@@ -73,6 +73,7 @@ tests = testGroup "Metadata" [
           MetadataNodeDefinition (MetadataNodeID 0) [ Just $ MDValue $ ConstantOperand (C.Int 32 1) ]
          ]
     let s = "; ModuleID = '<string>'\n\
+            \source_filename = \"<string>\"\n\
```

Comment:
If this source filename shows up in serialized modules now, have we provided a way to specify the value?